ocp_resources/cluster_service_version.py

```diff
@@ -6,3 +8,14 @@ class ClusterServiceVersion(NamespacedResource):
 
     class Status(NamespacedResource.Status):
         INSTALLING = "Installing"
+
+    def get_dicts_from_examples(self):
```

Comment:
If this is specific to `alm-examples`, then the name should reflect that, otherwise, the name of the annotation should be a param to this function. Wdyt?

Reply:
should be called `get_alm_examples`

Reply:
Please rename to `get_alm_examples`